src/rashdf/plan.py

```diff
@@ -1,9 +1,79 @@
 from .geom import RasGeomHdf
-
+from .base import RasHdf
+from typing import Dict
 from geopandas import GeoDataFrame
 
 
-class RasPlanHdf(RasGeomHdf):
+class RasPlanHdf(RasHdf):
```

Comment:
Totally agree on this one and overlooked it in my review. If we have have solid evidence of geometry differences between Plan and Geometry HDFs then I'm all for changing the inheritance like this, @sray014. I'm not as much of an expert as @bates-j on the RAS HDF format but in the files I've perused I haven't been able to spot any obvious differences in the `Geometry` group between Geom and Plan HDFs. Have you seen otherwise, @sray014?

Reply:
I'm not as familiar with the RAS HDF format either but that makes sense, there's definitely some similarity in the schema between the two. Changed